testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/MyResourcesTest.java

```diff
@@ -131,48 +121,133 @@ public void setup() throws Exception {
         }
     }
 
+    private ResourceRepresentation createResource(AuthzClient authzClient, AuthorizationResource authorization, int i) {
+        ResourceRepresentation resource = new ResourceRepresentation();
+
+        resource.setOwnerManagedAccess(true);
+
+        try {
+            final byte[] content = new JWSInput(authzClient.obtainAccessToken("jdoe", PASSWORD).getToken()).getContent();
+            final AccessToken accessToken = JsonSerialization.readValue(content, AccessToken.class);
+            resource.setOwner(accessToken.getSubject());
+        }
+        catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+
+        resource.setName("Resource " + i);
+        resource.setDisplayName("Display Name " + i);
+        resource.setIconUri("Icon Uri " + i);
+        resource.addScope("Scope A", "Scope B", "Scope C", "Scope D");
+        resource.setUri("http://resourceServer.com/resources/" + i);
+
+        try (Response response1 = authorization.resources().create(resource)) {
+            resource.setId(response1.readEntity(ResourceRepresentation.class).getId());
+        }
+        return resource;
+    }
+
+    @Override
+    public void addTestUser() {
+        testUser = createUser("jdoe");
+    }
+
+    @Test
+    public void shouldDisplayTheResources() {
+        assertEquals(6, myResourcesPage.getResourcesListCount());
+
+        assertEquals("Resource 0", myResourcesPage.getCellText("name", 0));
+        assertEquals("Resource 1", myResourcesPage.getCellText("name", 1));
+        assertEquals("My Resource Server", myResourcesPage.getCellText("client", 0));
```

Comment:
Please add a simple assert to verify that link is also present here.
I believe there is some scenario when a link is not present at all, right? We should test that too.

Reply:
there should always be a link: https://github.com/keycloak/keycloak/blob/0b75961986ccb625fb55a9777fac29dc0315d729/themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/my-resources-page/ResourcesTable.tsx#L130

Reply:
What if the client doesn't have Base URL? ;)